foot.sh

```diff
@@ -0,0 +1,2 @@
+export SHELL="host-spawn $(flatpak-spawn --host getent passwd $USER | awk -F : '{ print $7 }')"
+/app/bin/foot --term=xterm-256color
```

Comment:
As it is, it will just ignore any arguments that can be potentially passed by user.